src/containers/ClassifierContainer.jsx

```diff
@@ -227,6 +230,31 @@ class ClassifierContainer extends React.Component {
             {this.state.popup}
           </Popup>
         }
+        
+        {console.log('='.repeat(80), '\n', this.props, this.state)}
```

Comment:
Should this log still be in here?

Reply:
Nyarp! No it shouldn't, removing now - thanks for the spot!